test/rpmlintrc/multiple/sample.rpmlintrc

```diff
@@ -0,0 +1 @@
+setBadness('suse-dbus-unauthorized-service', 0)
```

Comment:
Is this a valid test? We don't have this lint rule in master...

Reply:
It could've been just empty file, I copied just parsable line thats it.